CHANGELOG.md

```diff
@@ -14,7 +14,18 @@
         self.startNavigation(with: route, animated: false)
     #endif
     ```
-* Custom location snapping in the `RouteController` via the delegate
+* Only require background audio when using speech synthesis by @michaelkirk in https://github.com/maplibre/maplibre-navigation-ios/pull/64
+* Restore the ability to init NavVC without animations. by @michaelkirk in https://github.com/maplibre/maplibre-navigation-ios/pull/73
+* Remove unnecessary keyboard tracking.  by @michaelkirk in https://github.com/maplibre/maplibre-navigation-ios/pull/69
+* Fix crash in EndOfRouteViewController and restore its presentation by @michaelkirk in https://github.com/maplibre/maplibre-navigation-ios/pull/71
+* Fix retain cycles in RouteMapViewController by @michaelkirk in https://github.com/maplibre/maplibre-navigation-ios/pull/74
+* Add "Build Example Project" to CI by @michaelkirk in https://github.com/maplibre/maplibre-navigation-ios/pull/77
```

Comment:
I'm personally in favor of having the changelog only include entries which are relevant to users. If they want every single PR they can look at Github, otherwise the important stuff just gets lost in the "tweaked the CI script" and "fixed tests". That work is important too, but that it makes the changelog less useful to users.

(There are some inarguably useful ones added here  - mostly ones that I forgot to add 😞)

Reply:
I don't feel super strongly about this btw, if people want to include every single PR in the changelog, I will choose another hill to die on. 😆

Reply:
Yeah, I agree, a changelog should contain changes 👍

Reply:
I have no strong feelings about this either. My thoughts:
- Who decides what is truly important and what isn't? Everything is a change, and someone who has had broken tests for months may be super excited about that specific change
- More exposure for everyone who does a PR
- It's much easier than manual change log maintenance: github offers a button for this, while in the other case, we've had so many PRs that need to go through an additional "review loop" because someone forgot to update the changelog with their change, delaying progress, often by days or weeks.

You're welcome to dismiss this PR though if the manual changelog maintenance is preferred.

Reply:
I don't have any strong feelings about this either, all good, just wanted to mention my opinion :)